apps/condo/domains/common/utils/xss.js

```diff
@@ -0,0 +1,19 @@
+const _xss = require('xss')
+
+const whiteList = _xss.getDefaultWhiteList()
+whiteList.p.push('style')
+whiteList.span.push('style')
+whiteList.li.push('style')
+whiteList.a.push('style')
+
+const xss = new _xss.FilterXSS({
+    whiteList,
+    css: {
+        whiteList: {
+            'text-align': true,
+            'background-color': true,
+        },
+    },
+})
+
+module.exports = { xss }
```

Comment:
it's bettert to just export something like `sanitizeXss(html)`. It's help us to change the XSS chack implementation in a future

Reply:
And write some tests for this util.